src/home/home.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ColumnComponent from "../components/ColumnComponent";
+
+const Home = () => {
+    return (
+        <div>
+            <h1>Home page</h1>
+            <ColumnComponent title='this is string' />
+        </div>
+    );
+};
```

Comment:
you shouldn't have this HomePage again. Now I merged your homePage to master. So you have to do the following:
git fetch
then in webshtorm when you see list of branches click on remote master and merge it into your columnComponent branch.

with resolving conflicts